azkaban-common/src/main/java/azkaban/project/CronSchedule.java

```diff
@@ -26,7 +27,7 @@
  * It couldn't be changed once gets constructed.
  * It will be used to schedule a trigger.
  */
-public class CronSchedule {
```

Comment:
Why does this class need to be Serializable?

Reply:
it's needs to persisted by quartz.